### PR TITLE
Added new project called as "Object Detector"

### PR DESCRIPTION
**Pull request by NANDAGOPALNG**
### Requesting to submit a pull request to the PyVerse repository.

---

#### Issue Title
**Please enter the title of the issue related to your pull request.**  
*[Code Addition Request]: Add new Project called as "Object Detector"*

- [x] I have provided the issue title.

---

#### Info about the Related Issue
**What's the goal of the project?**  
*Describe the aim of the project.*

- [ ] I have described the aim of the project.

---

#### Name
**Please mention your name.**  
*Nanda Gopal.D*

- [x] I have provided my name.

---

#### GitHub ID
**Please mention your GitHub ID.**  
*https://github.com/NANDAGOPALNG*

- [x] I have provided my GitHub ID.

---

#### Email ID
**Please mention your email ID for further communication.**  
*nandagopalng2004@gamil.com*

- [x] I have provided my email ID.

---

#### Identify Yourself
**Mention in which program you are contributing (e.g., WoB, GSSOC, SSOC, SWOC).**  
*GSSOC-ext*

- [x] I have mentioned my participant role.

---

#### Closes
**Enter the issue number that will be closed through this PR.**  
*Closes: #1075*

- [x] I have provided the issue number.

---

#### Describe the Add-ons or Changes You've Made
**Give a clear description of what you have added or modified.**  
*Describe your changes here.*

- [ ] I have described my changes.

---

#### Type of Change
**Select the type of change:**  
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

---

#### How Has This Been Tested?
**Describe how your changes have been tested.**  
*Describe your testing process here.*

- [ ] I have described my testing process.

---

#### Checklist
**Please confirm the following:**  
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

File: Generative-AI/Object Detector/README.md
Comment (UTSAVS26) [inline Generative-AI/Object Detector/README.md:1]: Do proper formatting. Alongwith proper links.

File: Generative-AI/Object Detector/app.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+def detect_object(image):
+    raw_image = image
+    output = object_detector(raw_image)
+    processed_image = draw_bounding_boxes(raw_image, output)
+    return processed_image
+
+demo = gr.Interface(fn=detect_object,
+                    inputs=[gr.Image(label="Select Image",type="pil")],
+                    outputs=[gr.Image(label="Processed Image", type="pil")],
+                    title="@GenAILearniverse Project 6: Object Detector",
```
Comment (UTSAVS26) [inline Generative-AI/Object Detector/app.py:78]: Why is it named Project 6?

Comment (NANDAGOPALNG) [reply Generative-AI/Object Detector/app.py:78]: This was my 6th project which was thaught to me in my collage club during a work shop conducted on Gen AI

**Review by ruhi47 — APPROVED** [review]
Excellent work!

**Pull request merged**